src/containers/ExpandedPostModal.js

```diff
@@ -10,19 +10,28 @@ import Post from '../components/Post'
 export default class ExpandedPostModal extends React.Component {
   static propTypes = {
     dispatch: func,
+    socket: object,
     commentId: string,
     post: object
   }
 
   componentDidMount () {
-    const { commentId, post: { id }, dispatch } = this.props
+    const { commentId, post: { id, numComments }, dispatch, socket } = this.props
     dispatch(fetchComments(id, {offset: 3}))
     .then(() => {
       if (commentId) {
         const wrapper = document.getElementById(modalWrapperCSSId)
         scrollToComment(commentId, wrapper)
       }
     })
+    socket.on(`p/${id}/comment_added`, function (){
+      dispatch(fetchComments(id, {offset: numComments, refresh: true}))
```

Comment:
@levity could you comment on this approach of just using a message to notify that there's a new comment, and then getting the client side to fetch it? We could pass it down, but it feels like that would be weird, because then we'd have to append it into the comments array or something?